PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                            <locus from="41r" to="48v"/>
+                                        </item>
+                                        <item xml:id="q7" n="7">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="49r" to="56v"/>
+                                        </item>
+                                        <item xml:id="q8" n="8">
+                                         <dim unit="leaf">6</dim>
+                                            <locus from="57r" to="62v"/>
+                                        </item>
+                                       </list>
+                                </collation>
+                                <condition key="good">The beginning and end of <ref target="#ms_i1"></ref> and the binding are missing. 
+                                    The other text block is in a good condition.</condition>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="12">
+                                     <ab type="pricking">Primary pricks are sometimes visible, horizontal pricks are not visible.</ab>
+                                    <ab type="ruling">Ruling is visible.</ab>
+                                    <ab type="ruling">The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                    <locus from="1r" to="62r"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The manuscript can be dated to the fourteenth or fifteenth century based on palaeographic features. For example, the letter መ shows a tendency for loop separations, ሎ demonstrates the
+                                    vowel marker with a linking line, ጵ exhibits the form used before the sixteenth century. Additionally, the word እግዚአብሔር is 
+                                    written as a single word. Overall, the manuscript displays the standard orthography 
+                                    of medieval Ethiopic. </desc>
+                               <seg type="ink">Black, red.</seg>
+                                <date notBefore="1400" notAfter="1500"> </date>
+                               </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                               <item xml:id="e1">
+                                   <locus target="#44v #45"></locus>
+                                   <desc>The name <persName ref="PRS14538AskalaGiyorgis"/> is written in supplications over erasure, 
```

Comment:
`role="owner"`

since the original name had been erased we cannot judge how close the replacement was, he surely was not the original donor, or no erasure would have been necessary